build/azure-pipelines/linux/Dockerfile

```diff
@@ -1,7 +1,11 @@
-#Download base image ubuntu 22.04
-FROM mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
+# Download base image ubuntu 20.04
+FROM mcr.microsoft.com/mirror/docker/library/ubuntu:20.04
```

Comment:
Do you still need to downgrade ubuntu with the rest of the changes?

Reply:
no, I don't think we need to downgrade, but this is what I tested with and it is only restoring to the OS version we've used in the past several releases (so it should be "known good").  I'm thinking we can upgrade to v22 at some point we when have an actual need for that.